setup.py

```diff
@@ -12,5 +12,9 @@
     url="http://www.maxlab.lu.se",
     packages=['svgsynoptic2'],
     include_package_data=True,
+    install_requires=["python-fandango", 
+                      "python-taurus",
+                      "python-pytango",
```

Comment:
This is the RPM package name. It should be the Python package name (fpm will automatically add the python prefix when creating the RPM): fandango, taurus, pytango